src/azul/service/manifest_service.py

```diff
@@ -492,6 +492,11 @@ def _get_cached_manifest_file_name(self,
             else:
                 raise e
         else:
+            lm = response['LastModified']
```

Comment:
@nadove-ucsc, (optionally) consider adding a `FIXME: …` above this line with a ref to the `azul-private#96` tix as extra precaution and for increased visibility.